src/cim/components/helpers/versions.ts

```diff
@@ -2,7 +2,7 @@ import _ from 'lodash';
 import { AgentClusterInstallK8sResource, ClusterImageSetK8sResource } from '../../types';
 import { OpenshiftVersionOptionType, OpenshiftVersion } from '../../../common';
 
-const getVersion = (releaseImage = '') => {
+export const getVersion = (releaseImage = '') => {
```

Comment:
Since you're exporting the function it should have a better name
```suggestion
export const getVersionFromReleaseImage = (releaseImage = '') => {
```